custom-plugins/grafana-worldmap-panel-msupply/src/worldmap.ts

```diff
@@ -141,17 +141,14 @@ export default class WorldMap {
   updateCircles(data) {
     const selectedFacilityName = _.find(this.ctrl.vars, elem => {
       return elem.name === this.ctrl.panel.linkedVariable;
-    }).current.value;
-
+    })?.current.value;
     data.forEach(dataPoint => {
       if (!dataPoint.locationName) {
         return;
       }
-
       const circle = _.find(this.circles, cir => {
-        return cir.options.location === dataPoint.key;
+        return cir?.options?.location === dataPoint.key;
```

Comment:
What does the `?` do exactly? It's something to do with not returning `null` values, is that right?

Is this just for Typescript, or Javascript too?